app/react/src/server/cra-config.js

```diff
@@ -10,22 +10,37 @@ const cssModuleExtensions = ['.module.css', '.module.scss', '.module.sass'];
 const typeScriptExtensions = ['.ts', '.tsx'];
 
 let reactScriptsPath;
+
 export function getReactScriptsPath({ noCache } = {}) {
   if (reactScriptsPath && !noCache) return reactScriptsPath;
-  const appDirectory = fs.realpathSync(process.cwd());
-  const reactScriptsScriptPath = fs.realpathSync(
-    path.join(appDirectory, '/node_modules/.bin/react-scripts')
-  );
-  reactScriptsPath = path.join(reactScriptsScriptPath, '../..');
+
+  try {
+    const appDirectory = fs.realpathSync(process.cwd());
+    const reactScriptsScriptPath = fs.realpathSync(
+      path.join(appDirectory, '/node_modules/.bin/react-scripts')
+    );
+
+    reactScriptsPath = path.join(reactScriptsScriptPath, '../..');
+    const scriptsPkgJson = path.join(reactScriptsPath, 'package.json');
+
+    // eslint-disable-next-line import/no-dynamic-require,global-require
+    require(scriptsPkgJson);
```

Comment:
Is this just to test if the package exists? If so, could you use `fs` for that?

Reply:
sure